### PR TITLE
OCPBUGS-28967:  'Oh no somthing went wrong' shown on Image Manifest Vulnerability page after create IMV via CL

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
### Before:

Screenshot 2024-02-05 at 5 20 03 PM

### After:

Screenshot 2024-02-06 at 12 53 10 PM

Screenshot 2024-02-21 at 8 34 05 AM

### CODE REVIEW

**Comment by cyril-ui-developer** [comment]
@jhadvig Updated, PTAL.

**Comment by cyril-ui-developer** [comment]
@jhadvig Updated, PTAL.

File: frontend/packages/container-security/src/components/image-manifest-vuln.tsx [hunk 1]
```diff
@@ -78,10 +79,11 @@ export const ImageManifestVulnDetails: React.FC<ImageManifestVulnDetailsProps> =
                 obj={props.obj}
                 path="obj.spec.manifest"
               >
-                <ExternalLink
-                  text={shortenHash(props.obj.spec.manifest)}
-                  href={quayURLFor(props.obj)}
-                />
+                {queryURL ? (
```
[the remark this answers is not in the document]
Comment (cyril-ui-developer) [reply frontend/packages/container-security/src/components/image-manifest-vuln.tsx:82]: > The behaviour should be to render the DetailsItem only if there is value to render

@jhadvig Hmm, not sure if that is doable without duplicating the label.

[the remark this answers is not in the document]
Comment (jhadvig) [reply frontend/packages/container-security/src/components/image-manifest-vuln.tsx:82]: @cyril-ui-developer which label do you mean ?

[the remark this answers is not in the document]
Comment (jhadvig) [reply frontend/packages/container-security/src/components/image-manifest-vuln.tsx:82]: Sorry for the confusion, I mean we should be conditioning the `DetailsItem` component with `queryURL`.

**Pull request merged**